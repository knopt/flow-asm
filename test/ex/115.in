3
9
-5.680642804392988
-4.836295075648332 -1.5690879475158042 -4.559335546543624 
-4.600911564189065 -8.468796402794336 7.355165748274313 
0.7782530611717604 8.968185200626731 3.585615181879845 
1.6356266941737978 1.3691814820362254 -0.7026648312445101 
2.2001962412350586 0.046162805969888865 -8.256411421358347 
-3.8812527754262423 -3.931886727044345 -4.3693769877461435 
-9.236227023462236 8.29709349810766 4.542361349860212 
-5.47270321059176 2.6936601477653035 9.414153948773205 
-2.532967687936674 0.5293655217109077 7.004822765464688 

-5.889687025645534 0.06457922057686538 -7.395140427127109 -8.311063648444641 0.6596497537722286 -8.089824384831886 5.574157526475727 -4.017858132751004 -6.339558173542034 
