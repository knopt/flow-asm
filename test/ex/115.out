-28.029597631421282 73.40823491650286 -67.01963941953622 
-37.12979225601174 -241.85303936947733 227.8734490240681 
128.5823491622515 316.4475917640975 57.02344881283846 
116.64727568990324 -37.15987259764317 -53.11260144401541 
166.8697925758827 16.173218667422116 -199.65699551469356 
-63.608758184671835 -100.9879775490709 -132.434949744829 
-181.3236554336798 356.5993281380556 64.80354542200396 
-66.88648795979717 127.0073682726137 145.76927298069847 
44.225632827623734 39.726410495294104 54.59320741528993 
